platform/security/servlet/security-servlet-logout/src/main/java/org/codice/ddf/security/servlet/logout/LogoutServiceImpl.java

```diff
@@ -57,7 +57,7 @@ public String getActionProviders(HttpServletRequest request, HttpServletResponse
     Object token =
         ((SecurityTokenHolder) session.getAttribute(SecurityConstants.SECURITY_TOKEN_KEY))
             .getPrincipals();
-    SessionToken sessionToken = new SessionToken(token);
+    SessionToken sessionToken = new SessionToken(token, session.getId(), "127.0.0.1");
```

Comment:
❓ Isn't this backwards from what is in WebSSOFilter?

Reply:
Yes I missed this one 🤦‍♀

Reply:
This would prevent these mistakes in the future: https://github.com/codice/ddf/pull/5942#discussion_r400570978